site/pages/dashboard.page.tsx

```diff
@@ -215,6 +226,26 @@ const Dashboard = ({
                             </>
                         ),
                     },
+                    {
+                        tabHeader: "Recently closed",
+                        content: (
+                            <>
+                                <Table
+                                    caption={{ text: "Closed disruptions", size: "l" }}
+                                    columns={["ID", "Summary", "Affected dates"]}
```

Comment:
says Dates effected on the wireframe - clarify with chris

Reply:
Figma designs will be updated as confirmed by Chris